packages/legend-application-marketplace/src/application/LegendMarketplaceApplicationConfig.ts

```diff
@@ -107,6 +109,8 @@ export class LegendMarketplaceApplicationConfig extends LegendApplicationConfig
         `Can't configure application: 'marketplace.url' field is missing or empty`,
       ),
     );
+    this.marketplaceSubscriptionUrl =
```

Comment:
if this is required to run the app we should do `LegendApplicationConfig.resolveAbsoluteUrl`